test/parallel/test-stdin-script-child.js

```diff
@@ -22,11 +22,11 @@ child.stderr.on('data', function(c) {
 });
 
 child.on('close', function(c) {
```

Comment:
Yes, I also think it needs some better documentation. I'll add my thoughts and experiences to the opened issue. I just checked the function out in the source code and will try to make the changes in a bit.